joomla/joomla-cms
# [4] Template. Allow manifest tag inheritable = 0 or empty

### Summary of Changes
Allow tag `<inheritable>0</inheritable>` in `templateDetails.xml`. Use `empty()` instead of `!isset()`.

### Testing Instructions
- Install test template https://github.com/GHSVS-de/GHSVSThings/raw/master/relaterquatch/nonamespacedummynoparent.zip (stolen from @HLeithner).
- The `templateDetails.xml` looks like this (see `<inheritable>0</inheritable>`):

```
<?xml version="1.0" encoding="utf-8"?>
<extension version="4.3" type="template">
	<name>nonamespacedummynoparent</name>
	<creationDate>2022-10-21</creationDate>
	<author>Me</author>
	<authorEmail>joomla@invalid.domain</authorEmail>
	<authorUrl>http://example.com</authorUrl>
	<copyright>Copyright (C) All rights reserved.</copyright>
	<version>1.0</version>
	<description>Test template for namespaces no namespace with no parent</description>
	<inheritable>0</inheritable>
	<files folder="templates/nonamespacedummynoparent">
		<folder>helpers</folder>
		<filename>index.php</filename>
		<filename>templateDetails.xml</filename>
	</files>

	<positions>
		<position>mainmenu</position>
	</positions>
</extension>
```
- In backend go to templates (not styles).
- Click template link "Nonamespacedummynoparent Details and Files"

### Actual result BEFORE applying this Pull Request
A error message:

`The template folder is not writable. Some features may not work.`

![grafik](https://user-images.githubusercontent.com/20780646/204543109-213038b4-a763-45d5-b7f9-26e0672ae59b.png)

### Expected result AFTER applying this Pull Request
No error message

### Comment
My build procedure for templates fills tags like `inheritable` using placeholders like `<inheritable>{{inheritable}}</inheritable>`. So I think values `0`, `1`, `empty` should be accepted without a misleading error message.

### Link to documentations
Please select:
